tests/iscsi/iscsi_client.pm

```diff
@@ -108,7 +108,7 @@ sub run {
     zypper_call("in open-iscsi yast2-iscsi-client");
     mutex_wait('iscsi_target_ready', undef, 'Target configuration in progress!');
     record_info 'Target Ready!', 'iSCSI target is configured, start initiator configuration';
-    apply_workaround_bsc1207157() if (is_sle('=15-SP3'));
+    apply_workaround_bsc1207157() if ((is_sle('=15-SP3')) || (is_sle('=15-SP5')));
```

Comment:
this bug is a duplicate, please use bsc#1206132

Reply:
Updated, VR is running. http://openqa.nue.suse.com/tests/10690238#details